app/assets/javascripts/discourse/tests/unit/lib/i18n-test.js

```diff
@@ -218,6 +213,20 @@ module("Unit | Utility | i18n", function (hooks) {
     assert.strictEqual(I18n.t("word_count", { count: 3 }), "3 words");
     assert.strictEqual(I18n.t("word_count", { count: 10 }), "10 words");
     assert.strictEqual(I18n.t("word_count", { count: 100 }), "100 words");
+
+    I18n.locale = "ja";
+    assert.strictEqual(
+      I18n.t("topic_stat_sentence_week", { count: 0 }),
+      "先週、新しいトピックが 0 件投稿されました。"
+    );
+    assert.strictEqual(
+      I18n.t("topic_stat_sentence_week", { count: 1 }),
```

Comment:
This allows to be sure we have the proper pluralization rules loaded. In Japanese (like most Asian languages), there is no plural, the rule returns `other` for everything. When the rule is not loaded properly, it returns the rule for the English language as a fallback. This in turn doesn’t work properly for `one` since it doesn’t exist in the Japanese translations. So the library in that specific case will output the English version for that key.